gatekeeper/client/client.go

```diff
@@ -0,0 +1,67 @@
+// Package client provides the Gatekeeper bootstrap API
+package client
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/manifoldco/torus-cli/gatekeeper/apitypes"
+	"github.com/manifoldco/torus-cli/registry"
+)
+
+const (
+	gatekeeperAPIVersion = "v0"
+	clientTimeout        = time.Minute
+)
+
+type clientRoundTripper struct {
+	registry.DefaultRequestDoer // TODO: Could be a generic RequestDoer?
```

Comment:
I was thinking instead of using the same `registry.DefaultRequestDoer` it would make sense to move that to a generic package, since this client isn't talking to the registry. I'll update the comment in the code too